app/src/main/java/ru/prometeydev/movie/FragmentMoviesDetails.kt

```diff
@@ -5,10 +5,18 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.TextView
+import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import ru.prometeydev.movie.data.adapters.ActorsAdapter
+import ru.prometeydev.movie.data.domain.ActorsDataSource
 
 class FragmentMoviesDetails : Fragment() {
```

Comment:
Could you add specific film details when you click a specific card on the movies list?

Reply:
This is I would say extra and might be out of scoup. But would make the app really complete and accurate.

Reply:
I thought about it, but I do not have data on the rest of the films to use when switching

Reply:
I made changes. Added a message display when clicking on a card if there is no additional information about the movie, also made the transfer of data to a fragment with details, via bundle, became more realistic. No data is lost when the screen is rotated.

Reply:
Great!